src/AppBundle/GitHooks/Action/CheckAssetsAction.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AppBundle\GitHooks\Action;
+
+use AppBundle\GitHooks\Action\Action;
+use CaptainHook\App\Config;
+use CaptainHook\App\Console\IO;
+use SebastianFeldmann\Git\Repository;
+
+final class CheckAssetsAction extends Action
+{
+    protected $defaultErrorMessage = "I'm sorry, Dave. I'm afraid I can't do that. Please build production assets before committing changes";
+
+    public function execute(Config $config, IO $io, Repository $repository, Config\Action $action): void
+    {
+        if (!$this->isEnabled($action)) {
```

Comment:
We could maybe extract this to parent `Action::execute()` and rename all other `execute` methods to `protected function doExecute(...);` in order not to repeat the `>isEnabled()` call in all of the actions?

Reply:
Done in c6217287